sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## 7.0.2 (Unreleased)
 
+**New Features**
+
+* Updated the following so that they accept lists and single instances of dict representations for corresponding strongly-typed object arguments:
+  - `update_queue`, `update_topic`, `update_subscription`, and `update_rule` on `ServiceBusAdministrationClient` accept dict representations of `QueueProperties`, `TopicProperties`, `SubscriptionProperties`, and `RuleProperties`, respectively.
+  - `send_messages` and `schedule_messages` on both sync and async versions of `ServiceBusSender` accept a list of or single instance of dict representations of `ServiceBusMessage`.
+  - `add_message` on `ServiceBusMessageBatch` now accepts a dict representation of `ServiceBusMessage`.
+  - Note: This is ongoing work and is the first step in supporting the above as respresentation of type `typing.Mapping`.
+  - Note: Thanks to bradleydamato for their large contribution to this.
```

Comment:
I'll probably do it like this following our normal pattern :P
```suggestion
* Updated the following methods so that lists and single instances of dict representations are accepted for corresponding strongly-typed object arguments (PR #14807, thanks @bradleydamato):
  - `update_queue`, `update_topic`, `update_subscription`, and `update_rule` on `ServiceBusAdministrationClient` accept dict representations of `QueueProperties`, `TopicProperties`, `SubscriptionProperties`, and `RuleProperties`, respectively.
  - `send_messages` and `schedule_messages` on both sync and async versions of `ServiceBusSender` accept a list of or single instance of dict representations of `ServiceBusMessage`.
  - `add_message` on `ServiceBusMessageBatch` now accepts a dict representation of `ServiceBusMessage`.
  - Note: This is ongoing work and is the first step in supporting the above as respresentation of type `typing.Mapping`.
```